README.md

```diff
@@ -59,3 +59,177 @@ Kilo Code is a direct fork from Roo Code, and also includes the following featur
 ## Extension Development
 
 For details on building and developing the extension, see [DEVELOPMENT.md](/DEVELOPMENT.md)
+
+## Contributors
+
+Thanks to all the contributors who help make Kilo Code better!
+
+<table>
+  <tr>
+    <td align="center">
+      <a href="https://github.com/mrubens">
+        <img src="https://avatars.githubusercontent.com/u/2600?v=4?size=100" width="100" height="100" alt="mrubens" style="border-radius: 50%;" />
+        <br />
+        <sub><b>mrubens</b></sub>
+      </a>
+    </td>    <td align="center">
+      <a href="https://github.com/chrarnoldus">
+        <img src="https://avatars.githubusercontent.com/u/12196001?v=4?size=100" width="100" height="100" alt="chrarnoldus" style="border-radius: 50%;" />
+        <br />
+        <sub><b>chrarnoldus</b></sub>
+      </a>
```

Comment:
Hmm, since this is a markdown file, I feel like this should probably be markdown and not html? Does this render right?